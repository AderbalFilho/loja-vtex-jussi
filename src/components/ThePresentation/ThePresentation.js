import React from 'react';

import { JSection } from '@ui';

import './ThePresentation.scss';

function ThePresentation() {
  return (
    <JSection classes="the-presentation" color="green">
      JÃ¼ssi
    </JSection>
  );
}

export default ThePresentation;
