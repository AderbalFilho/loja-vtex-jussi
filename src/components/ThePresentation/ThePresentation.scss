.the-presentation {
  padding: 40px;
}

.the-presentation__content {
  display: flex;
  margin: 0 auto;
  max-width: 1024px;
}

.the-presentation__left-content {
  margin-right: 24px;
  max-width: 451px;
  position: relative;
}

.the-presentation__slashes {
  color: var(--white);
  position: absolute;
  margin: 0 0 0 -87px;
}

.the-presentation__title {
  margin: 0 0 16px 0;
}

.the-presentation__subtitle {
  margin: 0 0 32px 0;
  max-width: 329px;
}

.the-presentation__solutions-button {
  display: block;
  max-width: 198px;
}

.the-presentation__right-content {
  display: flex;
}
