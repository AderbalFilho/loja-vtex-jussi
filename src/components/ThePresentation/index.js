import ThePresentation from './ThePresentation';

export default ThePresentation;
