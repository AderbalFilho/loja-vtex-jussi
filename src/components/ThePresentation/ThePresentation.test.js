import React from 'react';
import TestRenderer from 'react-test-renderer';

import ThePresentation from './ThePresentation';

describe('ThePresentation', () => {
  it('matches snapshot', () => {
    const testRenderer = TestRenderer.create(<ThePresentation />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
