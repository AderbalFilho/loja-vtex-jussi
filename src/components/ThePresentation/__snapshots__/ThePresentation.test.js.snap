// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThePresentation matches snapshot 1`] = `
<section
  className="j-section j-section--green the-presentation"
>
  <div
    className="the-presentation__content"
  >
    <div
      className="the-presentation__left-content"
    >
      <h1
        className="j-title j-title--extra-large the-presentation__slashes"
      >
        //
      </h1>
      <div>
        <h1
          className="j-title j-title--extra-large the-presentation__title"
        >
          Criamos lojas que vendem mais.
        </h1>
        <p
          className="j-text j-text--black j-text--normal the-presentation__subtitle"
        >
          A Jüssi é especialista na criação de lojas usando a plataforma VTEX. Precisa criar sua loja ou migrar de plataforma?
        </p>
        <a
          className="j-button j-button--back-transparent j-button--border-black j-button--black j-button--rad-normal j-button--link the-presentation__solutions-button"
          href="https://jussi.com.br/services.html"
          rel="noreferrer noopener"
          target="_blank"
        >
          Veja nossas soluções
        </a>
      </div>
    </div>
    <div
      className="the-presentation__right-content"
    >
      <div
        className="j-card j-card--rad-normal j-card--shadow j-card--size-small the-presentation-card the-presentation__offer1"
      >
        <div
          className="the-presentation-card__img-container"
        >
          <img
            alt="Refrigerador"
            className="the-presentation-card__img"
            src=""
          />
        </div>
        <a
          className="j-button j-button--back-transparent j-button--border-pink j-button--pink j-button--rad-circle j-button--link the-presentation-card__button"
          href="#"
          rel="noreferrer noopener"
          target="_blank"
        >
          Mais Detalhes
        </a>
      </div>
      <div
        className="j-card j-card--rad-normal j-card--shadow j-card--size-small the-presentation-card the-presentation__offer2"
      >
        <div
          className="the-presentation-card__img-container"
        >
          <img
            alt="Batedeira"
            className="the-presentation-card__img"
            src=""
          />
        </div>
        <button
          className="j-button j-button--back-transparent j-button--border-pink j-button--pink j-button--rad-circle the-presentation-card__button"
          onClick={[Function]}
        >
          Comprar em 12x
        </button>
      </div>
      <div
        className="j-card j-card--rad-normal j-card--shadow j-card--size-small the-presentation-card the-presentation__offer3"
      >
        <div
          className="the-presentation-card__img-container"
        >
          <img
            alt="Whisky"
            className="the-presentation-card__img"
            src=""
          />
        </div>
        <button
          className="j-button j-button--back-transparent j-button--border-pink j-button--pink j-button--rad-circle the-presentation-card__button"
          onClick={[Function]}
        >
          Adicionar a sacola
        </button>
      </div>
    </div>
  </div>
</section>
`;
